src/libparms/db/parmsdb.cpp

```diff
@@ -525,6 +525,27 @@
 #define UPDATE_SELF_RESTARTER_SERVER_REQUEST_ID "update_self_restarter_server_time"
 #define UPDATE_SELF_RESTARTER_SERVER_REQUEST "UPDATE self_restarter SET lastServerRestart=?1;"
 
+//
```

Comment:
This file is huge. Would it be possible to create a new header and a new cpp file to introduce the functions that would be called here?

Reply:
I split the code relating to this PR in new `.h` and `.cpp` files. I will create a Jira ticket to plan the same refactor for the rest of the file.